src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/TypeReference.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.signalr;
+
+import java.lang.reflect.Type;
+import java.lang.reflect.ParameterizedType;
+
+public class TypeReference<T> {
+
+    private final Type type;
+
+    protected TypeReference() {
+        Type superclass = getClass().getGenericSuperclass();
+        if (superclass instanceof Class) {
```

Comment:
Haha, the highlighting here :D

Reply:
https://www.youtube.com/watch?v=-cmo6MRYf5g